docker-storage-setup.sh

```diff
@@ -888,6 +895,17 @@ else
   fi
 fi
 
+# Main Script
+
+if [ $# -gt 0 ]; then
+    if [ "$1" == "reset" ]; then
```

Comment:
We might want to keep this secret since it does not do it all, or we would have to at least document that you still need to rm -rf /var/lib/docker to get this to work.

We are just doing this in docker-storage-setup for a helper to atomic storage reset.